rsts/concepts/launchplans.rst

```diff
@@ -1,8 +1,24 @@
 .. _divedeep-launchplans:
   
 Launch plans
-============
-Launch plans are used to execute workflows. A workflow can have many launch plans, but an individual launch plan is always tied to a single, specific workflow. Once created, launch plans are easy to share and execute.
+=============
+Launch plans help execute workflows. A workflow can be associated with multiple launch plans, but an individual launch plan is always associated with a single, specific workflow. 
+
+Every workflow comes with a `default` launch plan. While creating a new workflow, this default launch plan is authored (in code). The default launch plan has the same name as that of the workflow. 
+
+.. note::
+   Users rarely interact with the default launch plan.
+
+A workflow can be associated with multiple launch plans and launch plan versions.
+A workflow version can be associated with multiple launch plans and their versions under different names.
+The above two statements can be combined to state that a worflow version can have multiple schedules associated with it, as long as these schedules exists as versions under different launch plan names.
+Suppose we have ``Workflow A`` in ``version 1``, ``LaunchPlans`` ``A`` and ``B`` in ``version 1``, and ``LaunchPlan`` ``C`` in ``version 2``, then:
+
+1. ``Workflow A`` can be associated with ``LaunchPlan A`` (version 1);
+2. ``Workflow A`` can be associated with ``LaunchPlan B`` (different launch plan name with same version);
+3. ``Workflow A`` can be associated with ``LaunchPlan C`` (different launch plan name and version).
+
```

Comment:
@SmritiSatyanV, can you add "A launch plan version can only ever be mapped to one WF version; meaning a launch plan version cannot be used twice. This is because part of what makes a new launch plan version is the mapping to the specific workflow version".